### ISSUE TITLE
cannot reproduce the results with sparsity=0.9 in cola dataset

### ISSUE THREAD

**Issue by bestfleer** [yuchaoli/PST#1]
Hello. I'm very interested in your work and glad to your released code. However, I meet problem when I reproduce the results with sparsity=0.9 in cola dataset with your code. The matthews_correlation only gets 0.25. My training setting follows your paper, which contains lr=5e-3, epoch=20/40. Can you offer some suggestions for my reproduce? Thanks for your help

**Comment by yuchaoli** [comment]
Sorry for the late reply. The learning rate should be ranged in {3e-5, 5e-5, 1e-4, 5e-4}, you set an incorrect lr=5e-3.
